templates/article_viewer.tmpl

```diff
@@ -8,6 +8,160 @@
 </head>
 <script>
     const root = {{ .API_ROOT }};
+    const authorUsername = {{ .authorUsername }};
+    const localUsername = {{ .localUsername }};
+
+    function postComment() {
+        const content = document.querySelector("input").value;
+        const data = {
+            "userid": {{ .userID }},  
+            "articleid": {{ .articleID }},
+            "parentid": null,
+            "content": content,
+        }
+        fetch("/comments", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(data)
+        })
+        .then(res => res.json())
+        .then(res => {
+            const e = document.createElement('p');
+            e.innerHTML = localUsername + ": " + res.content;
+            e.id = res.comment_id;
+            document.body.appendChild(e);
+
+            const reply = document.createElement('button');
+            reply.id = res.comment_id;
+            reply.onclick = function() {
+                createReply(this.id);
+            }
+            reply.innerHTML = "reply";
+            e.appendChild(reply);
+        })
+    }
+
+    function createReply(id) {
+        const replyInput = document.createElement('input');
+        replyInput.id = "replyInput";
+        replyInput.classList.add("commentReply");
+        document.getElementById(id).appendChild(replyInput);
+
+
+        const submitReply = document.createElement('button');
+        submitReply.innerHTML = "Submit";
+
+
+        const cancelButton = document.createElement('button');
+        cancelButton.innerHTML = "Cancel";
+        cancelButton.onclick = function() {
```

Comment:
can we wrap these three elements in a div, then delete that div? something like:
```
<div id="replyWrapper">
  <div id="replyInput">...</div>
  <div id="submitReply">...</div>
  <div id="cancelButton">...</div>
</div>
```
then to remove:
```
document.getElementById(replyWrapper).innerHTML = '';
document.getElementById(id).removeChild(replyWrapper);
```